.github/workflows/check-backport.yml

```diff
@@ -1,7 +1,10 @@
 name: Stable Backport Check
 on:
   issue_comment:
-  pull_request:
+    types:
+      - created
+      - edited
+  pull_request_target:
```

Comment:
I knew there was a reason I had that! cc @h1dden-da3m0n

Reply:
yeah, sorry :frowning_face: That one was my bad. (funny enough I also stumbled over this one recently, after I commented, and had to switch too :sweat: )